src/fernet/core.clj

```diff
@@ -72,7 +72,7 @@
           (> ts (+ now max-clock-skew)))
       (invalid-token))))
 
-(defn decrypt-token [key-material token
+(defn decrypt-token [key-material ^bytes token
```

Comment:
While I think this should be type-hinted as well, this might be best done with the other functions not directly affected by the PR.